### ISSUE TITLE
RFC: FrontendWorkflowForm is a hack and should be removed or heavily refactored.

### ISSUE THREAD

**Issue by silbinarywolf** [symbiote/silverstripe-advancedworkflow#299]
The current state of the "FrontendWorkflowForm" is that is has not been touched since 2012 (except by me) and does a bunch of quirky things that make routing hard to follow.

For backwards compatibility, we could just split it out into a "silverstripe-advancedworkflow-bc" branch for antiquated projects.

Moving forward, I'd prefer the underlying ObjectCreatorForm ideas presented here:
https://github.com/nyeholt/silverstripe-frontend-objects/issues/11

To also contain have WorkflowForm support in some sort of way.

**Comment by SilbinaryWolf** [comment]
Random thought: Perhaps the Form itself can become instead an Extension to add Workflow functionality to your own form.

**Comment by robbieaverill** [comment]
This RFC hasn't received any traction in two years, so I'm going to close it. Thanks for raising it though!
